src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
+
+## v0.13.4
+
+This is a bugfix release:
+
+* Fix issues in Azure cloud prepare:
+  * Machine set names are now based on region + uuid and limited to 20 characters to prevent issues with long cluster names.
+  * Machine set creation and deletion logic was updated to prevent creation of multiple gateway nodes.
+  * Image names are now retrieved from existing machine sets.
+* Fix issues in Service Discovery:
+  * Namespace is now included in EndpointSlice name.
+  * EndpointSlice service lookup is now done using labels.
+* The `subctl gather` command will now gather iptables logs for Calico and kindnet CNIs.
+* The `subctl cloud` commands now always return the appropriate status.
```

Comment:
It's based on this commit: https://github.com/submariner-io/subctl/commit/6aa8a806e8aabc96f8ceaa7bb2e3478a29d0d56e

Reply:
That was to make sure to end the internal status properly in all cases otherwise we may omit textual output. But I'm not sure there was an actual case where that was observed.  This one is sort of vague and won't make sense to an end user. We could say something like "_The `subctl cloud` commands now always outputs the appropriate text_" but I think we should just omit it.

Reply:
Then I guess we should also omit the previous one? I can send that.

Reply:
Which is the previous one?